Initialization/linux-initialization-2.md

```diff
@@ -442,9 +442,8 @@ And the `_AC` macro is defined in the [include/uapi/linux/const.h](https://elixi
 #define _AC(X,Y)	__AC(X,Y)
 #endif
 ```
-So, where `__PAGE_OFFSET` expands to `0xffff880000000000`.
+Where `__PAGE_OFFSET` expands to `0xffff880000000000`. But, why a virtual address can be translated to the physical address by simply subtracting `__PAGE_OFFSET`? The answer is in the [Documentation/x86/x86_64/mm.txt](https://elixir.bootlin.com/linux/v3.10-rc1/source/Documentation/x86/x86_64/mm.txt#L9) documentation: 
```

Comment:
s/But, why a virtual address can be translated to the physical address by simply subtracting `__PAGE_OFFSET`?
 /But, why is it possible to translate a virtual address to a physical address by subtracting `__PAGE_OFFSET`?